WordPress/src/main/java/org/wordpress/android/ui/posts/PromoDialog.kt

```diff
@@ -113,10 +113,14 @@ class PromoDialog : AppCompatDialogFragment() {
     }
 
     private fun initializeView(view: View) {
-        val image = view.findViewById<ImageView>(R.id.promo_dialog_image)
-        image.setImageResource(drawableResId)
         val imageContainer = view.findViewById<LinearLayout>(R.id.promo_dialog_image_container)
-        imageContainer.visibility = if (DisplayUtils.isLandscape(activity)) View.GONE else View.VISIBLE
+        if (drawableResId == -1) {
```

Comment:
this is the place I was referring to for using `UNDEFINED_RES_ID` (from https://github.com/wordpress-mobile/WordPress-Android/pull/9234/files#r257675909)